### PR TITLE
Left join

### PR DESCRIPTION
**Pull request by M-Ravenmoore**
- [ ] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [ ] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/left-join/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+can start a hashTable
+can hash a key
+can Add a key/value to your hashtable
+creates a linked list
+'Can successfully instantiate an empty tree
+Can successfully return a collection from all traversals
+
+## Solution
+<!-- Link to img of whiteboard of solution -->
+![whiteBoard]('./assets/CC-whiteboard.jpg');
```
Comment (adrianhuebner) [inline javascript/code-challenges/left-join/README.md:30]: This isn't linking up right

File: javascript/code-challenges/left-join/leftJoin.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const HashMap = require('./src/hashMap');
+
+const joinedMap = new HashMap(222);
+
+const mapArray = [];
+
+function leftJoin(mapArr, opt){
```
Comment (adrianhuebner) [inline javascript/code-challenges/left-join/leftJoin.js:9]: I would suggest changing the names of the parameters here to something like: hashTable1 and hashTable 2 to make it a little more clear
